app/src/Entity/Event.php

```diff
@@ -279,4 +279,9 @@ public function getTalks()
     {
         return $this->talks;
     }
+
+    public function getPath()
```

Comment:
I don't think getPath should be here. Entities just hold data. Getting a path of an entity is routing. Entities should have no knowledge of routing.  There is already  code that gets the path of an Event.

Reply:
OK, I'll look for the other code, remove this and update the code where it's used.

Reply:
See example here (line 5): https://github.com/phpsw/phpsw-ng/blob/2f87abe0a0fea1db087551bf5564a944e9eb5a1a/app/resources/views/events.twig.html

The eventUrl is in my lastest PR. Perhaps once that is through review you can merge it into this PR then you'll have the code you need?